pkgs/tools/system/sleuthkit/default.nix

```diff
@@ -17,17 +55,34 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
-  nativeBuildInputs = [ autoreconfHook ];
+  nativeBuildInputs = [ autoreconfHook openjdk perl ant rdeps ];
   buildInputs = [ libewf afflib openssl zlib ];
 
-  # Hack to fix the RPATH.
+  # Hack to fix the RPATH
   preFixup = "rm -rf */.libs";
 
-  meta = {
+  IVY_HOME = "/build/.ant";
```

Comment:
We can't hardcode /build like this. Can this be any temporary directory? If so just use ``$(mktemp -d)``.

Reply:
Oh right, thanks @SuperSandro2000 , that can indeed break build in many slightly different setups (like incomplete sandboxing).

Reply:
Since mktemp (and NIX_BUILD_TOP?) are shell expressions, not nix expressions, I had to define IVY_HOME and ant_args in a preUnpack and preBuild since they won't evaluate in a attribute assignment.
I ended up using NIX_BUILD_TOP.